contribs/drt/src/main/java/org/matsim/contrib/drt/routing/DrtStopFacilityImpl.java

```diff
@@ -27,28 +27,32 @@
 import org.matsim.api.core.v01.Identifiable;
 import org.matsim.api.core.v01.network.Link;
 import org.matsim.facilities.Facility;
+import org.matsim.utils.objectattributes.attributable.Attributable;
+import org.matsim.utils.objectattributes.attributable.Attributes;
 
 /**
  * @author Michal Maciejewski (michalm)
  */
 public class DrtStopFacilityImpl implements DrtStopFacility {
-	public static <F extends Identifiable<?> & Facility> DrtStopFacility createFromIdentifiableFacility(F facility) {
+	public static <F extends Identifiable<?> & Facility & Attributable> DrtStopFacility createFromIdentifiableFacility(F facility) {
 		return new DrtStopFacilityImpl(Id.create(facility.getId(), DrtStopFacility.class), facility.getLinkId(),
-				facility.getCoord());
+				facility.getCoord(), facility.getAttributes());
```

Comment:
Are facilities (practically) always Attributable? If that is not the case, maybe do `if facility instanceof Attributable` then `facility.getAttributes()` else `new Attributes()` ?

Reply:
Good point, the `TransitStopFacilities` are attributable, so when reading from the transit schedules file I was expecting that the `DrtStopFacility` takes over the attributes from the file. What other use cases are there? Also, the case you mention will not happen because the template signature of the function requires `Attributable`. If somebody tries to call `createFromIdentifiableFacility` with something that is now `Attributable`, it will be a syntax error. But maybe the function could be renamed to "IdentifiableAttributable" or just "createFromFacility" ?

Reply:
> What other use cases are there? 

No idea. I do not work a lot on building scenarios. That's why I am asking :-)

> Also, the case you mention will not happen because the template signature of the function requires Attributable.

Yes, but that's only after your change. Before it wasn't required.

So I would say if it is very unlikely that facilities will not be `Attributable`, let's merge. Otherwise, rather than requiring `Attributable`, take the alternative approach with `instanceof Attributable`.

Reply:
As for the method name, `createFromFacility` sounds good.